torch/_prims/__init__.py

```diff
@@ -1633,6 +1633,23 @@ def _convert_element_type_nvfuser(fd: Any, a: Tensor, dtype: torch.dtype) -> Ten
     doc=_convert_element_type_doc,
 )
 
+# NOTE: _to_dtype
+# This private op casts the input to the desired type while preserving its stride
+# permutation, unlike .to(dtype) which will create a tensor with contiguous strides
```

Comment:
to() is supposed to preserve strides (that's why its memory format defaults to preserve_format). File a bug?

Reply:
Good call -- https://github.com/pytorch/pytorch/issues/77600